python/kserve/kserve/model.py

```diff
@@ -44,6 +44,24 @@
 EXPLAINER_V2_URL_FORMAT = "{0}://{1}/v2/models/{2}/explain"
 
 
+class BaseKserveModel(ABC):
```

Comment:
Kserve word is redudant here as it is already in the kserve package

Reply:
`BaseModel` was my first choice, but it conflicts with pydantic's `BaseModel` in to many places. We wouldn't want to deal with aliases. Do you have another suggestion?

Reply:
I see, this is probably ok I think the base model shouldn’t be used in a lot of the places. Can you help fix the case of KServe ?